script.js

```diff
@@ -196,4 +196,25 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     errorMsg.classList.add('display-content');
   }
-});
\ No newline at end of file
+});
+
+// form localStorage Setting
+
+document.getElementById('submit').addEventListener('click', () => {
+  const formDataObj = {
+    name: document.getElementById('formName').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('comment').value,
+  };
+  const formObj = JSON.stringify(formDataObj);
+  localStorage.setItem('formData', formObj);
+});
+document.getElementById('reset').addEventListener('click', () => {
+  localStorage.removeItem('formData');
+});
+
+const formObject = localStorage.getItem('formData');
+const objectData = JSON.parse(formObject);
+document.getElementById('formName').value = objectData.name;
+document.getElementById('email').value = objectData.email;
+document.getElementById('comment').value = objectData.message;
```

Comment:
- [ ] When the user loads the page, the code checks if there are any saved data in the `localstorage` and if so the input fields are pre-filled with the data. You could do this by wrapping your already written code here with an `if statement` when the page loads.
```
// For instance:

window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```

Reply:
solved thanks